runners/katacoda/templates/intro.md

```diff
@@ -0,0 +1,7 @@
+<%= description; %>
+
+<% if(tutorialPath){ %>
+The definition of each step of this tutorial can be found at https://github.com/devonfw-tutorials/katacoda-scenarios/tree/master/<%= tutorialPath; %>
+
+Feel free to fix any errors you find yourself. You just need to create a pull request to the katacoda-scenarios repository with the changed files.
```

Comment:
Please provide a direct link on 'pull request' to the already available document in how this is done. I think it Is in the devonfw contribution guide